applications/CoSimulationApplication/python_scripts/coupling_operations/convert_distributed_values_to_point.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import print_function, absolute_import, division  # makes these scripts backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics as KM
+
+# Importing the base class
+from KratosMultiphysics.CoSimulationApplication.base_classes.co_simulation_coupling_operation import CoSimulationCouplingOperation
+
+class ConvertDistributedValuesToPoint(CoSimulationCouplingOperation):
+
+    def __init__(self, settings, solver_wrappers):
+        super(ComputeNormalsOperation, self).__init__(settings)
+        solver_name = self.settings["solver"].GetString()
+        data_name = self.settings["data_name"].GetString()
+        self.interface_data = solver_wrappers[solver_name].GetInterfaceData(data_name)
+
+        if self.interface_data.is_scalar_variable:
+            self.intermediate_variable = KM.KratosGlobals.GetVariable(self.settings["intermediate_variable_scalar"].GetString())
+        else:
+            self.intermediate_variable = KM.KratosGlobals.GetVariable(self.settings["intermediate_variable_scalar"].GetString())
+
+    def Execute(self):
+        KM.VariableRedistributionUtility.ConvertDistributedValuesToPoint(
+            self.interface_data.GetModelPart(),
+            self.interface_data.variable,
+            self.intermediate_variable)
```

Comment:
I think the order of the last two arguments of this function should be changed.
for example in a Dirichlet-Neumann coupling, this function would be used on the structure's FSI interface to convert distributed tractions from an intermediate auxiliary variable to POINT_LOAD.